core-galleon-pack/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,126 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2014, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<build xmlns="urn:wildfly:feature-pack-build:2.0">
+    <default-packages>
+        <package name="modules.all"/>
+        <package name="docs"/>
+    </default-packages>
+    <package-schemas>
+        <group name="org.wildfly.core"/>
+    </package-schemas>
+
+    <config model="standalone">
+        <props>
+            <prop name="config.branch-is-batch" value="true"/>
+            <prop name="--admin-only" value=""/>
+            <prop name="--internal-empty-config" value=""/>
+            <prop name="--internal-remove-config" value=""/>
+        </props>
+        <packages>
+            <package name="product.conf" optional="true"/>
+            <package name="misc.standalone"/>
+            <package name="org.jboss.as.standalone.main"/>
```

Comment:
We discussed this earlier in chat, but I really believe that the default module slot (main) should not be included in package names.